AP_Randomizer/apworld/rules_expert.py

```diff
@@ -89,11 +110,37 @@ def __init__(self, world) -> None:
                 and (
                     self.has_gem(state)
                     or self.get_kicks(state, 2)),
+            # "Underbelly Little Guy -> Upper Bailey": lambda state: True,  # technically already true because obscure
             "Underbelly Hole -> Underbelly Main Lower": lambda state:
                 self.has_slide(state),
         }
 
         location_clauses = {
+            "Empty Bailey - Cheese Bell": lambda state:
+                self.has_slide(state)
+                and (
+                    self.kick_or_plunge(state, 2)
```

Comment:
you can do slide 1 kick fairly easily

Reply:
just got that myself too 👍 
```suggestion
                    self.kick_or_plunge(state, 1)
```

Reply:
I haven't done it but it would definitely be hard/maybe require fp. if it's possible probably lunatic?

Reply:
```suggestion
                    self.get_kicks(state, 1)
```
we can always add it to lunatic later